jcommune-model/src/main/resources/org/jtalks/jcommune/model/entity/Group.hbm.xml

```diff
@@ -67,7 +67,15 @@
       <![CDATA[from Group where id in (:ids)]]>
   </query>
 
-    <query name="selectGroupsWithUserCount">
-        <![CDATA[select new org.jtalks.jcommune.model.dto.GroupAdministrationDto(g.id, g.name, g.description, g.users.size) from Group g order by g.name ASC]]>
-    </query>
+  <query name="selectGroupsWithUserCount">
+      <![CDATA[select new org.jtalks.jcommune.model.dto.GroupAdministrationDto(g.id, g.name, g.description, g.users.size) from Group g order by g.name ASC]]>
+  </query>
+
+  <query name="getGroupNameById">
+    <![CDATA[select new java.lang.String(g.name) from Group g where g.id = :id]]>
```

Comment:
It's not necessary to have separate query to get group name. Group isn't very complex object, so you can just get group by id and then get group name.